requirements.txt

```diff
@@ -1,4 +1,6 @@
 lxml
+numpy
+scipy
```

Comment:
It works on linux, you'll probably need some prerequisites installed like gcc and gfortran, so I probably should add some apt-gets to travis config, this is impossible to know until you try :)

webstruct definitely uses scipy, tests fail unless it's installed and it is not installed with your old requirements (though should be). The reason is scipy and scikit-learn don't set `install_requires` properly, webstruct also follows this dirty tradition. 

I ran this on my Linux laptop, so travis config should be fine besides some apt-gets.